docs/user/dashboard/dashboard-troubleshooting.asciidoc

```diff
@@ -0,0 +1,69 @@
+[[dashboard-troubleshooting]]
+== Dashboard and visualizations troubleshooting
+++++
+<titleabbrev>Troubleshooting</titleabbrev>
+++++
+
+Refer to the troubleshooting information to find solutions to common dashboard and visualization issues.
```

Comment:
Maybe start the sentence at "Find solutions" so the first part doesn't repeat the title.